webrtc.html

```diff
@@ -648,6 +657,39 @@ <h4><dfn>RTCRtcpMuxPolicy</dfn> Enum</h4>
           </div>
         </div>
       </section>
+      <section>
+        <h4><dfn>RTCAdaptivePtime</dfn> Enum</h4>
+        <div>
+          <pre class="idl">enum RTCAdaptivePtime {
+  "off",
+  "on",
+  "default"
```

Comment:
Suggestion for use of MUST/SHOULD which have specific meaning in spec and is upper-cased: When adaptivePtime is on, the User Agent SHOULD adapt the frame length based on network conditions and when it is off, the User Agent MUST NOT adapt the frame length based on network conditions.

Reply:
I'd suggest not overusing 2119 language. When adaptivePtime is off, the UA MUST use the ptime negotiated via SDP. When it is on, it may (lowercase!) vary within the envelope negotiated in the SDP

Reply:
1. I agree with fippo about using "may" rather than "must" in the on-case.
2. Why should "may" be lowercase in the on-case, fippo?
3. Do we want to explicitly say what behavior is expected when adaptivePtime is simply not provided? I.e. is it up to the UA to decide if it defaults to on/off, or must it default to off?

Reply:
I've made changes. Please take a look.

Reply:
OK I'm now convinced "MAY" is better than "SHOULD" considering how optional it is and totally up to the UA about its strategy. Don't know about "MAY" vs "may" though.

Reply:
the "may" does not limit behaviour, see section 6 in 2119. Use them sparingly ;-)

Reply:
Hi fippo@. You're commenting on an out of date revision.